staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -429,7 +429,7 @@ func (s *GenericAPIServer) PrepareRun() preparedGenericAPIServer {
 //           |                                                       |
 //    ShutdownInitiated (shutdownInitiatedCh)                        |
 //           |                                                       |
-// (ShutdownDelayDuration)                                    (PreShutdownHooks)
+// (ShutdownDelayDuration)                               (PreShutdownHooksStopped)
```

Comment:
The convention I was using was basically that inside `()` I was marking something that is happening.

The signals themselves are marked without () (and if there is something in`()` after the signal, it's in alias in the code.

So I think you need to:
- leave this line as it was (basically signalling that preshutdown hooks are running at this point)
- switch line 424 to:
`PreShutdownHooksStopped (preShutdownHooksHasStoppedCh`

Reply:
fixed